pixart.js

```diff
@@ -1 +1,99 @@
+var $setColor = $('#set-color');
+var $input = $('input');
+var $brush = $('.brush').eq(0);
+var $body = $('body');
+var color = $brush.css('background-color');
+var colorSwatchColors = [color];
+var colorSwatchDivs = [];
+var swatchSize = 3;
+// var $swatchContainer = $('.swatch-container');
+updateSwatch();
+$setColor.on('click', setColor);
+// $brush.on('click', function() {
+//   color = $brush.css('background-color');
+// });
+// removed since chrome does by default
+// $input.on('keyup', function(evt) {
+//   evt.preventDefault();
+//   if(evt.keyCode === 13) {
+//     setColor(evt);
+//   }
+// });
 
+function setColor(evt) {
+  evt.preventDefault();
+    if (color !== $input.val()){
+      color = $input.val();
+      var oldColor = $brush.css('background-color');
+      $brush.css({
+        'background-color': color
+      });
+      var newColor = $brush.css('background-color');
+      if (oldColor !== newColor) {
+        updateColorSwatchColors();
+        updateSwatch();
+      }
+    }
+  $input.val('');
+}
+
+function createCanvas(num) {
+  for(var i = 0; i < num; i++) {
+    var $div = $('<div/>');
+    $div.toggleClass('square canvas');
+    $div.on('mouseover', function(evt){
+      $(this).css({
+        'background': color
+      });
+    });
+    $body.append($div);
+  }
+}
+
+createCanvas(8000);
```

Comment:
If you're going to call a function as soon as you declare it, you don't really need to have that function to begin with.

If you're feeling fancy, you could use an "IIFE" -- an "immediately-invoked function expression":

```
(function sayHello(){
  console.log("hello");
}());
```

You can't call `sayHello` anywhere else, and it's run as soon as it's defined. These are useful for visually tidying code, and also it prevents any variables defined inside the IIFE from cluttering up the global scope.